src/services/ui/src/api/useGetUser.test.ts

```diff
@@ -48,9 +48,26 @@ const mockUserAttr = ({ isCms, error }: { isCms?: boolean; error?: boolean }) =>
     );
   });
 
+const mockCurrentAuthenticatedUser = vi.fn((options = {}) => {
+  return new Promise((resolve, reject) => {
+    // You can decide how to use the options object, if needed, for your tests.
+    // If you want to simulate an error, you could set a flag on `options` and check it here.
+    if (options.error) {
+      reject(
+        new Error(
+          "useGetUser > mockCurrentAuthenticatedUser: Expected error thrown by test."
+        )
+      );
+    } else {
+      resolve({ username: "051ee598-f107-417b-af00-1dfe8bb6484c" });
+    }
+  });
+});
+
```

Comment:
You don't need this big block, you'll want to add the following to the above resolved array of user attributes (line 45): 

```typescript
{ Name: "username", Value: "0000aaaa-0000-00aa-0a0a-aaaaaa000000" }
// I usually remove any unique IDs even if they're for test user accounts
```